src/test/scala/com/goyeau/kubernetes/client/api/NamespacesApiTest.scala

```diff
@@ -11,63 +11,68 @@ import io.k8s.api.core.v1.{Namespace, NamespaceList}
 import io.k8s.apimachinery.pkg.apis.meta.v1.ObjectMeta
 import org.http4s.Status
 import org.http4s.client.UnexpectedStatus
-import org.scalatest.flatspec.AnyFlatSpec
 import org.scalatest.OptionValues
+import org.scalatest.flatspec.AnyFlatSpec
 import org.scalatest.matchers.should.Matchers
+import scala.concurrent.duration._
 
-import scala.concurrent.ExecutionContext
+class NamespacesApiTest
```

Comment:
test cases here create own unique namespace to not clash when they are running concurrently.